client/src/modules/prices/prices.service.js

```diff
@@ -72,5 +73,10 @@ function PriceListService(Api) {
       .then(service.util.unwrapHttpResponse);
   }
 
+  function download(params) {
+    const url = service.url.concat('download/list');
+    return service.$http.post(url, params)
```

Comment:
It looks like this will send a `POST` request, but the download list will receive a `GET` request.  If you are running into bugs, this might be where it is coming from.